src/services/item/plugins/invitation/schema.ts

```diff
@@ -159,3 +159,22 @@ export const sendOne = {
     },
   },
 };
+
+export const enroll = {
+  tags: ['itemMemberships'],
+  summary: 'Create an item membership for the logged in user if there is an Item Login',
+  description: `Create an item membership on the item with the given ID for the logged in user. 
+    The item be associated with an Item Login.`,
```

Comment:
```suggestion
    The item is associated with an Item Login.`,
```
not sure what it means with the remaining of the sentence tho? You want to add that it is allowed if the item has an item login right?